### DISCUSSION TITLE
In operator [Jun 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2509] [Discord Threads]
> 🙋‍♂️ **KVY**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### You can use list api with in operator  
#  
> 🙋‍♂️ **KVY**  
#### is it?  
![image0](https://cdn.discordapp.com/attachments/988790847237668924/988814139478667304/unknown.png)  
#  
> 💁 **o1lab**  
#### yes  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/988790847237668924)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
